### PR TITLE
🎉 Add the map view

### PR DESCRIPTION
**Pull request by mamadOuologuem**
image

### CODE REVIEW

File: src/components/MapView/Layers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  bboxsr: 3857,
+  imagesr: 3857,
+  width: 256,
+  height: 256,
+  srs: 'EPSG:3857',
+  bbox: '{bbox-epsg-3857}',
+})
+  .map((value, key) => `${key}=${value}`)
+  .join('&')
+  .value();
```
Comment (bencpeters) [inline src/components/MapView/Layers/index.tsx]: This seems like reinventing the wheel, and could be dangerous without escaping URI components.

You could use Node's [url library](https://www.npmjs.com/package/url)

File: .gitignore [hunk 2]
```diff
@@ -21,26 +21,6 @@
...
-# misc
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
```
[the remark this answers is not in the document]
Comment (mamadOuologuem) [reply .gitignore]: I remove the duplicate line

File: package.json [hunk 3]
```diff
@@ -37,6 +37,7 @@
     "@typescript-eslint/eslint-plugin": "^2.23.0",
     "@typescript-eslint/parser": "^2.25.0",
     "babel-eslint": "10.0.3",
+    "env-cmd": "^10.1.0",
```
[the remark this answers is not in the document]
Comment (mamadOuologuem) [reply package.json]: I don't know why but dotenv wasn't working when I was following this example https://github.com/motdotla/dotenv/tree/master/examples/typescript

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (bencpeters) [reply .gitignore]: Haha gotcha! I had the top of the collapsed when I reviewed, I didn't see the first 20 lines of the file 😊

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (bencpeters) [reply package.json]: Create React App has [some environment loading built-in](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env) - can you try this?

**Pull request merged**